src/sirtextstyle.c

```diff
@@ -203,15 +205,57 @@ bool _sir_setcolormode(sir_colormode mode) {
         /* when the color mode changes, it's necessary to regenerate the text styles
          * we're holding. for example in the case of downgrading color modes, the
          * styles in memory could be incompatible with the new mode. */
-        if (!_sir_resettextstyles()) {
+        if (!_sir_resettextstyles()) { // GCOVR_EXCL_START
```

Comment:
💀

Reply:
💀

Reply:
Easy to break this too. Simply make `_sir_formatstyle()` fail.

Reply:
Will remove for now